pyccel/codegen/printing/ccode.py

```diff
@@ -267,6 +267,7 @@ def __init__(self, parser, settings=None):
         self._additional_imports = set(['stdlib'])
         self._parser = parser
         self._additional_code = ''
+        self._tmp_var_tobereturned = None
```

Comment:
This new name is not clear and seems to indicate the opposite use to what is implemented. I suggest `_unused_return_variable`